client/components/tooltip/style.scss

```
@@ -3,6 +3,11 @@
 		// ensures that the element needed for position calculations isn't included in the DOM layout
 		display: contents;
 
+		// fixes a UI bug where the tooltip can shift neighboring elements when it appears
```

Minor nitpick – should our CSS comments be sentences with full stop? I thought we had a linter enforcing this like PHP / JS. Not a blocker just curious what our standard is.

Side note (also nitpick, not a blocker) – I find it clearer when css comments state the positive outcome/purpose of the rule, rather than define in terms of a bad behaviour that's fixed. Not sure what that would look like for this one.

---

Yeh, the linter enforces this for PHP.

Suggestion with the positive outcome/purpose:
```suggestion
		// Ensure that the tooltip doesn't shift neighboring elements when it appears.
```